www-client/firefox/firefox-9999.ebuild

```diff
@@ -31,9 +31,11 @@ ASM_DEPEND=">=dev-lang/yasm-1.1"
 RDEPEND="
 	>=dev-libs/nspr-4.13.1
 	>=dev-libs/nss-3.28.3
+	>=dev-util/cbindgen-0.6.4
```

Comment:
This doesn't build. Is there 0.6.3 intended?

Reply:
Fixed in new rebase-amend